### PR TITLE
feat(calendar-view): [calendar-view] Adapting to the SMB themes

### PR DESCRIPTION
**Pull request by Youyou-smiles**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Summary by CodeRabbit

- **New Features**
	- Introduced a new theme configuration for the calendar view, enhancing customization options.
	- Added new styles and CSS variables for improved theming and responsiveness in the calendar view component.
- **Improvements**
	- Enhanced styling capabilities with new CSS variables for better customization and theming.
	- Updated existing styles for improved consistency and responsiveness across the calendar component.
- **Configuration Changes**
	- Updated theme configuration to include the calendar view and removed obsolete components.

### CODE REVIEW

File: packages/theme/src/calendar-view/smb-theme.js [hunk 1]
```diff
@@ -19,10 +19,10 @@ export const tinyCalendarViewSmbTheme = {
   'ti-calendar-view-radio-active-inner-hover': 'var(--ti-common-color-bg-white-normal)',
   'ti-calendar-view-radio-button-inner': '#ebebeb',
   'ti-calendar-view-header-main-text-color': 'var(--ti-common-color-text-weaken)',
-  'ti-calendar-view-tooltip-title-font-size': 'var(--ti-common-font-size-2)',
-  'ti-calendar-view-tooltip-title-font-weight': 'var(--ti-common-font-weight-bold)',
-  'ti-calendar-view-tooltip-content-color': 'var(--ti-common-color-text-primary)',
-  'ti-calendar-view-tooltip-date-color': 'var(--ti-common-color-text-secondary)',
+  'ti-tooltip-title-font-size': 'var(--ti-common-font-size-2)',
```
Comment (chenxi-20) [inline packages/theme/src/calendar-view/smb-theme.js]: 命名改为ti-tooltip开头的变量会全局污染tooltip的变量，建议保持原样，或改为简洁一点的变量，如ti-calendar-view-tip-xxx

Comment (Youyou-smiles) [reply packages/theme/src/calendar-view/smb-theme.js]: > 命名改为ti-tooltip开头的变量会全局污染tooltip的变量，建议保持原样，或改为简洁一点的变量，如ti-calendar-view-tip-xxx

嗯。我这就去改。

File: packages/theme/src/calendar-view/vars.less [hunk 2]
```diff
@@ -27,10 +27,10 @@
     --ti-calendar-view-day-selected-svg-right: -1px;
     --ti-calendar-view-day-selected-svg-top: -6px;
     // 提示
-    --ti-calendar-view-tooltip-title-font-size: var(--ti-common-font-size-base);
-    --ti-calendar-view-tooltip-title-font-weight: var(--ti-common-font-weight-5);
-    --ti-calendar-view-tooltip-date-color: #8d959e;
-    --ti-calendar-view-tooltip-content-color: #8d959e;
+    --ti-tooltip-title-font-size: var(--ti-common-font-size-base);
+    --ti-tooltip-title-font-weight: var(--ti-common-font-weight-5);
+    --ti-tooltip-date-color: #8d959e;
```
Comment (zzcr) [inline packages/theme/src/calendar-view/vars.less]: 这里面怎么会出现tooltip的组件级变量呢？

File: examples/sites/demos/pc/app/calendar-view/calendar-event.spec.ts [hunk 3]
```diff
@@ -3,27 +3,27 @@ import { test, expect } from '@playwright/test'
 test('事件', async ({ page }) => {
   page.on('pageerror', (exception) => expect(exception).toBeNull())
   await page.goto('calendar-view#calendar-event')
-  const timeInput = page.locator('.tiny-filter-box').locator('.value')
-  const timeBtn = page.getByText('2023 年 05 月')
+  const timeInput = page.locator('.pc-demo-container').locator('.tiny-input__inner')
```
Comment (zzcr) [inline examples/sites/demos/pc/app/calendar-view/calendar-event.spec.ts]: 这里建议使用#calendar-event选择器

File: packages/theme/src/calendar-view/vars.less (hunk 2 above)
Comment (Youyou-smiles) [reply packages/theme/src/calendar-view/vars.less]: > 这里面怎么会出现tooltip的组件级变量呢？

这里有个弹窗，样式的权重没有处理好，我这就去处理。

File: examples/sites/demos/pc/app/calendar-view/calendar-event.spec.ts (hunk 3 above)
Comment (Youyou-smiles) [reply examples/sites/demos/pc/app/calendar-view/calendar-event.spec.ts]: > 这里建议使用#calendar-event选择器

好的。

**Review by zzcr — APPROVED** [review]

**Pull request merged**